docs/infrastructure.md

```diff
@@ -172,279 +170,213 @@ When you launch the graph backend for the first time,
-                graphdb:disable-sameAs "true" ;
-                graphdb:check-for-inconsistencies "false" ;
-
-                # Indexing
-                graphdb:entity-id-size "32" ;
-                graphdb:enable-context-index "false" ;
-                graphdb:enablePredicateList "true" ;
-                graphdb:enable-fts-index "false" ;
-                graphdb:fts-indexes ("default" "iri") ;
-                graphdb:fts-string-literals-index "default" ;
-                graphdb:fts-iris-index "none" ;
-
-                # Queries and Updates
-                graphdb:query-timeout "0" ;
-                graphdb:throw-QueryEvaluationException-on-timeout "false" ;
-                graphdb:query-limit-results "0" ;
-
-                # Settable in the file but otherwise hidden in the UI and in the RDF4J console
-                graphdb:base-URL "http://example.org/owlim#" ;
-                graphdb:defaultNS "" ;
-                graphdb:imports "" ;
-                graphdb:repository-type "file-repository" ;
-                graphdb:storage-folder "storage" ;
-                graphdb:entity-index-size "10000000" ;
-                graphdb:in-memory-literal-properties "true" ;
-                graphdb:enable-literal-index "true" ;
-            ]
-        ].
-    ```
+        ??? example "Doing this manually with `curl`"
+
+            ```bash
+            curl -X PUT --header 'Content-Type: application/json' -d '
+            {"grantedAuthorities": ["WRITE_REPO_my_db","READ_REPO_my_db"]}' http://localhost:7200/rest/security/users/DBUSER -u "admin:NewAdminPassword"
+            ```
 
-    Then you can create a new graph db with the following command (replace "my_db" as needed):
+            - `"WRITE_REPO_my_db"`: Grants write permission.
+            - `"READ_REPO_my_db"`: Grants read permission.
 
+            Make sure you replace `my_db` with the name of the graph db you have just created. 
+
+    If the script has run all steps successfully, you should see:
     ```bash
-    curl -X PUT -u "admin:NewAdminPassword" http://localhost:7200/repositories/my_db --data-binary "@data-config.ttl" -H "Content-Type: application/x-turtle"
+    Done.
     ```
 
-    and add give our user access permission to the new resource:
+    You can now proceed to the section [Uploading data to the graph](#uploading-data-to-the-graph).
+
+=== "Stardog"
+
+    **Note: Stardog has been deprecated as a supported Neurobagel graph backend.**
+
+## Updating your graph backend configuration
+
+### Updating existing database user permissions
 
+!!! tip
+    You can verify a user's settings at any time with the following:
     ```bash
-    curl -X PUT --header 'Content-Type: application/json' -d '
-    {"grantedAuthorities": ["WRITE_REPO_my_db","READ_REPO_my_db"]}'  http://localhost:7200/rest/security/users/DBUSER -u "admin:NewAdminPassword"
+    curl -u "admin:NewAdminPassword" http://localhost:7200/rest/security/users/DBUSER
```

Comment:
Move that below the first paragraph - shouldn't be the first thing the user sees / should not appear without context.

Reply:
I would move it under 

> Of note, in GraphDB, there is no straightforward REST API call to update a user's database access permissions without replacing the list of their existing database permissions ("grantedAuthorities") entirely.